agent/src/test/scala/za/co/absa/atum/agent/dispatcher/ConsoleDispatcherTest.scala

```diff
@@ -0,0 +1,102 @@
+//    val dispatcher = new ConsoleDispatcher()
+//    val additionalData = mock[AdditionalDataSubmitDTO]
+//    assert(dispatcher.saveAdditionalData(additionalData) == ())
+//  }
+
+//  test("saveCheckpoint should not throw exception") {
+//    val dispatcher = new ConsoleDispatcher()
+//    val checkpoint = mock[CheckpointDTO]
+//    try {
+//      dispatcher.saveCheckpoint(checkpoint)
+//    } catch {
+//      case e: Exception => fail("saveCheckpoint threw an exception", e)
+//    }
+//  }
+//
+//  test("saveAdditionalData should not throw exception") {
+//    val dispatcher = new ConsoleDispatcher()
+//    val additionalData = mock[AdditionalDataSubmitDTO]
+//    try {
+//      dispatcher.saveAdditionalData(additionalData)
+//    } catch {
+//      case e: Exception => fail("saveAdditionalData threw an exception", e)
+//    }
+//  }
+//
+//}
+
+class ConsoleDispatcherTest extends AnyFunSuite with MockitoSugar {
+
+//  private[agent] def toSeqPartitionDTO(atumPartitions: AtumPartitions): PartitioningDTO = {
+//    atumPartitions.map { case (key, value) => PartitionDTO(key, value) }.toSeq
+//  }
+
+//  test("createPartitioning should return AtumContextDTO") {
+//    val dispatcher = new ConsoleDispatcher()
+//    val atumPartitions =  AtumPartitions("abc" -> "def")
+//    val partitioningDTO = AtumPartitions.toSeqPartitionDTO(atumPartitions)
+//    val partitioning = PartitioningSubmitDTO(partitioningDTO, None, "testAuthor")
+//    val result = dispatcher.createPartitioning(partitioning)
+//    assert(result.isInstanceOf[AtumContextDTO])
+//    assert(result.partitioning == "testPartitioning")
+//  }
+
+//  test("createPartitioning should return AtumContextDTO") {
+//    val dispatcher = mock[ConsoleDispatcher]
+//    val atumPartitions =  AtumPartitions("abc" -> "def")
+//    val partitioningDTO = toSeqPartitionDTO(atumPartitions)
+//    val partitioning = PartitioningSubmitDTO(partitioningDTO, None, "testAuthor")
+//    when(dispatcher.createPartitioning(partitioning))
+//      .thenReturn(AtumContextDTO(partitioningDTO, Set.empty, AdditionalDataDTO(additionalData = Map.empty)))
+//
+//    val result = dispatcher.createPartitioning(partitioning)
+//    assert(result.isInstanceOf[AtumContextDTO])
+//    assert(result.partitioning == "testPartitioning")
+//  }
+
+//  test("saveCheckpoint should not throw exception") {
+//    val dispatcher = mock[ConsoleDispatcher]
+//    val checkpoint = mock[CheckpointDTO]
+//    try {
+//      dispatcher.saveCheckpoint(checkpoint)
+//    } catch {
+//      case e: Exception => fail("saveCheckpoint threw an exception", e)
+//    }
+//  }
+
+//  test("saveAdditionalData should not throw exception") {
+//    val dispatcher = new ConsoleDispatcher()
+//    val additionalData = mock[AdditionalDataSubmitDTO]
+//    try {
+//      dispatcher.saveAdditionalData(additionalData)
+//    } catch {
+//      case e: Exception => fail("saveAdditionalData threw an exception", e)
+//    }
+//  }
+}
```

Comment:
what is this one for? Lots of dead code

Reply:
Are you still planning to finish / remove it?

Reply:
I plan to remove that.